source/mainnet/conf.py

```diff
@@ -49,7 +49,9 @@
     "sphinx.ext.extlinks",
     # "sphinx.ext.imgconverter", # To support svg when targeting LaTeX
     "multidoc",
-    "sphinx_reredirects"
+    "sphinx_reredirects",
+    # "sphinx_rtd_dark_mode", # A bug in this prevents code blocks from displaying correctly in dark mode
+    "sphinx_copybutton"
 ]
 
```

Comment:
I would recommend adding

```
copybutton_prompt_text = "$"
copybutton_line_continuation_character = "\\"
```
here (and the same for testnet). This will mean that, for example, in the following 
![Screenshot from 2022-02-26 20-40-02](https://user-images.githubusercontent.com/105212/155856829-fce1f587-7b0a-4aaf-9254-cae6c711cd8a.png)

only the text
```
concordium-client config --help
```
is copied instead of both the input and output.

At least I think this is the most reasonable behaviour, don't you think?

Reply:
Yes. Have added both lines.